deployable/java/src/main/java/envtest/deployable/Snippets.java

```diff
@@ -63,15 +61,19 @@ public void simplelog(Map<String,String> args){
         Severity severity = getSeverity(severityString);
```

Comment:
Should the `Severity.valueOf(severityString)` used here instead? Perhaps `getSeverity()` could be rewritten to make `valueOf()` call and handle `IllegalArgumentException`.

Reply:
That may be worth doing if possible. I didn't realize the built in `valueOf()` already did the same thing